tvb_framework/tvb/interfaces/web/controllers/help/help_config.py

```diff
@@ -110,6 +110,7 @@ def _load_mappings(self):
         self._add_mapping(WebStructure.SECTION_CONNECTIVITY, WebStructure.SUB_SECTION_CONNECTIVITY, self.CONNECTIVITY_HELP_PAGE, "long-range-connectivity")
         self._add_mapping(WebStructure.SECTION_CONNECTIVITY, WebStructure.SUB_SECTION_LOCAL_CONNECTIVITY, self.CONNECTIVITY_HELP_PAGE, "local-connectivity")
         self._add_mapping(WebStructure.SECTION_CONNECTIVITY, WebStructure.SUB_SECTION_ALLEN, self.CONNECTIVITY_HELP_PAGE, "allen-connectome-downloader")
+        self._add_mapping(WebStructure.SECTION_CONNECTIVITY, WebStructure.SUB_SECTION_SIIBRA, self.CONNECTIVITY_HELP_PAGE, "siibra-connectivities")
```

Comment:
TODO: we need to write also in tvb-documentation a chapter to match this name

Reply:
Based on the recently added chapter in the documentation, I think this string should be: `siibra-connectivity-creator` from `Siibra Connectivity Creator` chapter title